docs/README.md

```diff
@@ -14,7 +14,12 @@ Playfff is a SRG micro service to serve extra datas to Play applications. Playff
 
 ## Installation
 
-A Java development environment with Maven is needed. To run the NodeJS server, `npm` and `python` (python 2) are needed as well.
+A Java 1.8 development environment with Maven is needed. To run the NodeJS server, `npm` is needed as well.
```

Comment:
The target bytecode is 1.8 but I've managed to compile and run the code with a JDK 17.

Reply:
@jboix Did you run tests as well? I'm in the middle of our CI which is working and my setup, not correctly setup.

Reply:
Good idea. Issue #78 opened. For a new PR. This one is too huge now.